test/esindex_test.py

```diff
@@ -36,6 +36,11 @@
 import os
 import unittest
 
+try:
+    from unittest import SkipTest
+except ImportError:
+    from unittest2 import SkipTest
```

Comment:
I see this pattern being copied at many places. Can we have a `luigi.unittest` that abstracts over this?